### ISSUE TITLE
Adams.java is using deprecated APIs - should be updated

### ISSUE THREAD

**Issue by paul-hammant** [GoogleCloudPlatform/google-cloud-datastore#174]
Builder.getMutableProperties()

The JavaDocs for that say:

```
/**
     * Use alternate mutation accessors instead.
     */
    @java.lang.Deprecated
```

There's a `setField` but I've no idea how to make a `Descriptors.FieldDescriptor` and searching for `FieldDescriptor` in this org on GH yields 90% Python and a few Java refs, but no illustration as to how to new one up, or make it by a builder/factory.

**Comment by wsh** [comment]
Hey--this is a little hard to parse without further context! Could you provide some more, please? (And note that what you've mentioned is about protobuf more than Datastore per se.)

**Comment by paul-hammant** [comment]
The example application for GoogleCloudPlatform uses deprecated APIs from another tech (Protobif for one, maybe others).  In Intellij deprecated method usage is indicated with lines through them:

![image](https://user-images.githubusercontent.com/82182/28238713-bb182074-6927-11e7-8212-ccc80c65437e.png)

The examples apps for google cloud should not use deprecated methods.

**Comment by wsh** [comment]
That context is super helpful. Just so you know, that example is based on our low-level API. You'll likely have a much better time with the google-cloud-java API: https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-datastore. Also take a look at https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine/guestbook-cloud-datastore for an example of how to use the API.
